builder/buffer.go

```diff
@@ -115,11 +115,25 @@ func (b *Buffer) WriteEscape(value string) {
 }
 
 func (b Buffer) escape(table, value string) string {
+	var escaped_table string
+	if table != "" {
+		parts := strings.Split(table, ".")
+		if len(parts) != 1 {
```

Comment:
maybe better like this instead?
```suggestion
		if strings.IndexByte(table, '.') >= 0 {
			parts := strings.Split(table, ".")
```

Reply:
Right, that part has temp code - I will change in next commit